nifi-nar-bundles/nifi-hubspot-bundle/nifi-hubspot-processors/src/main/java/org/apache/nifi/processors/hubspot/GetHubSpot.java

```diff
@@ -234,6 +237,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             FlowFile flowFile = session.create();
             flowFile = session.write(flowFile, parseHttpResponse(response, total, stateMap));
             if (total.get() > 0) {
+                session.putAttribute(flowFile, CoreAttributes.MIME_TYPE.key(), "application/json");
```

Comment:
Calling `session.putAttribute()` should update the reference to `flowFile`:
```suggestion
                flowFile = session.putAttribute(flowFile, CoreAttributes.MIME_TYPE.key(), "application/json");
```